Resources/public/js/alloyeditor/buttons/list.js

```diff
@@ -58,13 +58,66 @@ YUI.add('ez-alloyeditor-button-list', function (Y) {
             var css = "ae-button ez-ae-labeled-button" + this.getStateClasses();
 
             return (
-                React.createElement("button", {className: css, onClick: this._addList, tabIndex: this.props.tabIndex}, 
-                    React.createElement("span", {className: "ez-ae-icon ae-icon-bulleted-list"}), 
+                React.createElement("button", {className: css, onClick: this._addList, tabIndex: this.props.tabIndex},
+                    React.createElement("span", {className: "ez-ae-icon ae-icon-bulleted-list"}),
+                    React.createElement("p", {className: "ez-ae-label"}, Y.eZ.trans('list', {}, 'onlineeditor'))
+                )
+            );
+        },
+    });
+
+    /**
+     * The ButtonListIndexed component represents a button to add an ordered list (ol).
+     *
+     * @uses AlloyEditor.ButtonCommand
+     * @uses AlloyEditor.ButtonStateClasses
+     *
+     * @class eZ.AlloyEditor.ButtonList
+     */
+    ButtonList = React.createClass({displayName: "ButtonListIndexed",
```

Comment:
I think something is wrong with this generated js. I think this should be `ButtonListIndexed = ...` instead of `ButtonList = ...`